cli/src/commands/fix.rs

```diff
@@ -465,8 +471,15 @@ fn get_tools_config(ui: &mut Ui, config: &config::Config) -> Result<ToolsConfig,
         let mut tools: Vec<ToolConfig> = tools_table
             .into_iter()
             .sorted_by(|a, b| a.0.cmp(&b.0))
-            .map(|(_name, value)| -> Result<ToolConfig, CommandError> {
-                let tool: RawToolConfig = value.try_deserialize()?;
+            .filter_map(
+                |(_name, value): (_, _)| match value.try_deserialize::<RawToolConfig>() {
+                    // tool is disabled, omit it entirely
+                    Ok(tool) if !tool.enabled => None,
+                    result => Some(result),
+                },
+            )
```

Comment:
nit: you can do `.map(|(_name, value)| value.try_deserialize..).filter_ok(|tool| tool.enabled)`.

Reply:
Thank you, I was struggling trying to find the most readable way to do this.